starter-code/views/categories.ejs

```diff
@@ -1,3 +1,8 @@
-  <body>
-    <h1>Categories</h1>
-  </body>
+
+    <h1>Categories: </h1>
+    <ul>
+      <% response.forEach( (cat)=> { %>
+        <li><a href ="/?cat= %><%= cat %>">
+        <%= cat %></a></li> <%
```

Comment:
I had trouble parsing this line. The last `<%` would be better at the beginning of the next line.